airbyte-integrations/bases/connector-acceptance-test/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "connector-acceptance-test"
-version = "3.3.1"
+version = "3.3.3"
```

Comment:
Why skip a version? Does that mean that 3.3.2 release didn't work? I would assume it did as the tests were failing but I can't validate as I'm getting "403 forbidden" on test results

Reply:
I forgot to update pypproject.toml in last PR. (only changelog was updated)
Do I need to bump version to 3.3.2 and update docs instead?

>403 forbidden 

got the same error, at the same time I'm able to get the test report directly from Google Cloud Storage (seems like we use "wrong" link in github CI results.

Reply:
> Do I need to bump version to 3.3.2 and update docs instead?

I don't know because it seems like it is released to me since the tests were failing because of that. Or does the version number useless? [I don't see any new CATs version for a while now](https://hub.docker.com/r/airbyte/connector-acceptance-test/tags) so I'm not sure what is the new release process. @alafanechere can you chime in on that?

Reply:
I just confirmed with @alafanechere and we build the CAT before running them so the version here isn't used which explains our situation.